editor/src/clj/editor/model.clj

```diff
@@ -319,16 +345,21 @@
   (property vertex-attribute-overrides g/Any
             (default {})
             (dynamic visible (g/constantly false)))
+  (property vertex-attribute-overrides-value-sources g/Any
+            (default {})
+            (dynamic visible (g/constantly false)))
```

Comment:
Rename to `vertex-attribute-override-value-sources`?

TBH, I think it would be better to refactor our code a bit and add the `value-source` information to the entries in `vertex-attribute-overrides`. That would allow us to generalize the code in `graphics` to retain any orphaned values across all the resource types that make use of vertex attribute overrides.